modules/oadp-restic-timeouts.adoc

```diff
@@ -19,7 +19,7 @@ level=error msg="Error backing up item" backup=velero/monitoring error="timed ou
 ----
 
 .Procedure
-* Edit the values in the `spec.configuration.restic.timeout` block of the `DataProtectionApplication` CR manifest, as in the following example:
```

Comment:
As this is the first use of "CR" in this particular module, you need to exmand it as "custom resource (CR)".

I would also use "as the following example shows" or "as shown in the following example".